eZ/Publish/Core/FieldType/Keyword/KeywordStorage/Gateway.php

```diff
@@ -10,13 +10,14 @@
 
 use eZ\Publish\SPI\Persistence\Content\Field;
 use eZ\Publish\Core\FieldType\StorageGateway;
+use eZ\Publish\SPI\Persistence\Content\VersionInfo;
 
 abstract class Gateway extends StorageGateway
 {
     /**
      * @see \eZ\Publish\SPI\FieldType\FieldStorage::storeFieldData()
      */
-    abstract public function storeFieldData(Field $field, $contentTypeId);
+    abstract public function storeFieldData(Field $field, $contentTypeId, VersionInfo $versionInfo);
```

Comment:
Unnecessary. `VersionInfo` is not used within any implementation of `storeFieldData`.